ta/ta.mk

```diff
@@ -18,7 +18,10 @@ ifeq ($(CFG_TEE_TA_MALLOC_DEBUG),y)
 $(sm)-platform-cppflags += -DENABLE_MDBG=1
 endif
 
+# Config variables to be explicitly exported to the dev kit conf.mk
 ta-mk-file-export-vars-$(sm) += CFG_TA_FLOAT_SUPPORT
+ta-mk-file-export-vars-$(sm) += CFG_CACHE_API
```

Comment:
I noticed that this will cause `conf.mk` to contain `CFG_CACHE_API :=` which is quite useless. Not a big deal, but I'd say we need the following patch before this one:
```diff
diff --git a/ta/ta.mk b/ta/ta.mk
index 684e2eac4..7266ee1d3 100644
--- a/ta/ta.mk
+++ b/ta/ta.mk
@@ -127,7 +127,7 @@ $(conf-mk-file-export):
        $(q)echo sm := $$(sm-$(conf-mk-file-export)) > $$@.tmp
        $(q)echo sm-$$(sm-$(conf-mk-file-export)) := y >> $$@.tmp
        $(q)($$(foreach v, $$(ta-mk-file-export-vars-$$(sm-$(conf-mk-file-export))), \
-               echo $$(v) := $$($$(v));)) >> $$@.tmp
+               $$(if $$($$(v)),echo $$(v) := $$($$(v));,))) >> $$@.tmp
        $(q)echo '$$(ta-mk-file-export-add-$$(sm-$(conf-mk-file-export)))' | sed 's/_nl_ */\n/g' >> $$@.tmp
        $(q)$(call mv-if-changed,$$@.tmp,$$@)
 endef
```